server/src/test/java/org/elasticsearch/search/geo/GeoShapeQueryTests.java

```diff
@@ -77,14 +77,14 @@
     };
 
     private XContentBuilder createMapping() throws Exception {
-        XContentBuilder xcb = XContentFactory.jsonBuilder().startObject().startObject("type1")
```

Comment:
To check I'm not missing something, these changes are just for clean-up (and not required to get tests to pass)?

Reply:
I think this has bled in from a different branch (the PR for #47792) - will clean up